app/src/main/java/org/simple/clinic/login/applock/AppLockScreen.kt

```diff
@@ -101,6 +102,9 @@ class AppLockScreen(context: Context, attrs: AttributeSet) : RelativeLayout(cont
     }
   }
 
+  private fun forgotPinClicks() =
+      RxView.clicks(forgotPinButton).map { AppLockForgotPinClicked() }
```

Comment:
I have realized that for testing button clicks which simply perform action without any logic, like opening a dialog, popping the backstack, etc. does not offer any value. The error window is the same when the action is taken directly or through the controller. In both the cases, commenting out this line is sufficient to break functionality:

```kotlin
// ConfirmResetPinDialog.show(activity.supportFragmentManager)
```

I hope this was understandable. What do you think? Can we handle this click in the screen directly?

Reply:
TODO: See if these can be hooked into Heap in all cases if the events don't go through the controller.